test/checks/visibility/hidden-content.js

```diff
@@ -0,0 +1,48 @@
+describe('hidden content', function () {
+	'use strict';
+
+  var fixture = document.getElementById('fixture');
+
+	var checkContext = {
+		_data: null,
+		data: function (d) {
+			this._data = d;
+		}
+	};
+
+	afterEach(function () {
+		fixture.innerHTML = '';
+		checkContext._data = null;
+	});
+
+	it('was present on the page with aria-hidden', function () {
+    fixture.innerHTML = '<div id="target" aria-hidden="true"><p>Nothing here.</p></div>';
+		var node = fixture.querySelector('#target');
+		assert.isUndefined(checks['hidden-content'].evaluate.call(checkContext, node));
+	});
+
+	it('was present on the page with display: none', function () {
+    fixture.innerHTML = '<div id="target" style="display: none;"><p>Nothing here.</p></div>';
+		var node = fixture.querySelector('#target');
+		assert.isUndefined(checks['hidden-content'].evaluate.call(checkContext, node));
+	});
+
+	it('was present on the page with visibility: hidden', function () {
+    fixture.innerHTML = '<div id="target" style="visibility: hidden;"><p>Nothing here.</p></div>';
+		var node = fixture.querySelector('#target');
+		assert.isUndefined(checks['hidden-content'].evaluate.call(checkContext, node));
+	});
+
+	it('was present on the page within a span with aria-hidden', function () {
+    fixture.innerHTML = '<span id="target" aria-hidden="true">Nothing here.</span>';
```

Comment:
I don't think that test is specific enough to the scenarios where a developer might put `aria-hidden="true"` on an icon element inside of an anchor or button. I thought we'd discussed whitelisting those to minimize noise in Needs Review, since they are valid use cases. As written that test below still returns undefined.